app/view_models/admin/needs_attention_auctions_view_model.rb

```diff
@@ -12,7 +12,7 @@ def drafts_partial
   end
 
   def drafts
-    @_drafts ||= Auction.unpublished.map do |auction|
+    @_drafts ||= AuctionQuery.new.unpublished.map do |auction|
```

Comment:
Ah, got it. I just assumed that scopes and queries were handled by the Query object, since I assumed that the caller is abstracted away from knowing the internals of the Auction model (ie, whether something is a scope). But I also am way of too much abstraction for the sake of abstraction.

In short, I don't really have a particular axe to grind on this one, but we probably need to update our documentation about this.